### ISSUE TITLE
SmartCity-Accessibility #145 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#112]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 98 lines...]
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[11,52] cannot find symbol
  symbol:   class UnauthorizedAccessException
  location: package smartcity.accessibility.navigation.exception
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[120,56] cannot find symbol
  symbol:   class UnauthorizedAccessException
  location: class smartcity.accessibility.mapmanagement.Location
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[143,58] cannot find symbol
  symbol:   class UnauthorizedAccessException
  location: class smartcity.accessibility.mapmanagement.Location
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[122,35] cannot find symbol
  symbol:   class UnauthorizedAccessException
  location: class smartcity.accessibility.mapmanagement.Location
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[145,35] cannot find symbol
  symbol:   class UnauthorizedAccessException
  location: class smartcity.accessibility.mapmanagement.Location
[INFO] 5 errors 
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.861 s
[INFO] Finished at: 2016-12-28T14:14:09+00:00
[INFO] Final Memory: 25M/156M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:compile (default-compile) on project SmartCity-Accessibility: Compilation failure: Compilation failure:
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[11,52] cannot find symbol
[ERROR] symbol:   class UnauthorizedAccessException
[ERROR] location: package smartcity.accessibility.navigation.exception
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[120,56] cannot find symbol
[ERROR] symbol:   class UnauthorizedAccessException
[ERROR] location: class smartcity.accessibility.mapmanagement.Location
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[143,58] cannot find symbol
[ERROR] symbol:   class UnauthorizedAccessException
[ERROR] location: class smartcity.accessibility.mapmanagement.Location
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[122,35] cannot find symbol
[ERROR] symbol:   class UnauthorizedAccessException
[ERROR] location: class smartcity.accessibility.mapmanagement.Location
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/mapmanagement/Location.java:[145,35] cannot find symbol
[ERROR] symbol:   class UnauthorizedAccessException
[ERROR] location: class smartcity.accessibility.mapmanagement.Location
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [yaelAmitay] c6a676a9315ea4561828cff06d94f043fce4aaa5 - move exception class to the right package

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/145/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
